### PR TITLE
Known issues 3.10

### PR DESCRIPTION
**Pull request by ansoboleva**

### CODE REVIEW

File: 3.10/release-notes-known-issues310.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ ArangoSearch
...
+| **Date Added:** 2022-09-29 <br> **Component:** ArangoSearch <br> **Deployment Mode:** All <br> **Description:** When using an inverted index, some nested fields may be handled ambiguosly. <br> **Affected Versions:** 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [SEARCH-390](https://arangodb.atlassian.net/browse/SEARCH-390){:target="_blank"} (internal) |
+| **Date Added:** 2022-09-29 <br> **Component:** ArangoSearch <br> **Deployment Mode:** Cluster <br> **Description:** In a cluster deployment using the new View of the search-alias type, running the `arangodump`/`arangorestore` commands fails.<br> **Affected Versions:** 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [SEARCH-401](https://arangodb.atlassian.net/browse/SEARCH-401){:target="_blank"} (internal) |
```
Comment (Simran-B) [inline 3.10/release-notes-known-issues310.md:28]:
```suggestion
| **Date Added:** 2022-09-29 <br> **Component:** ArangoSearch <br> **Deployment Mode:** Cluster <br> **Description:** In a cluster deployment using the new View of the `search-alias` type, running the `arangodump`/`arangorestore` commands fails. <br> **Affected Versions:** 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [SEARCH-401](https://arangodb.atlassian.net/browse/SEARCH-401){:target="_blank"} (internal) |
```

File: 3.10/release-notes-known-issues310.md [hunk 2]
```diff
@@ -67,4 +70,6 @@ Other
...
\ No newline at end of file
+| **Date Added:** 2021-08-06 <br> **Component:** Installer <br> **Deployment Mode:** Single Server <br> **Description:** The Windows installer fails during database initialization with the error `failed to locate tzdata` if there are non-ASCII characters in the destination path. <br> **Affected Versions:** 3.8.x, 3.9.x, 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [BTS-531](https://arangodb.atlassian.net/browse/BTS-531){:target="_blank"} (internal) |
+| **Date Added:** 2022-09-29 <br> **Component:** Web UI <br> **Deployment Mode:** All <br> **Description:** In the Web UI for ArangoSearch Views, it's not possible to create a link to a collection, if the collection has only one letter in its name. <br> **Affected Versions:** 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [BTS-1008](https://arangodb.atlassian.net/browse/BTS-1008){:target="_blank"} (internal) |
```
Comment (Simran-B) [inline 3.10/release-notes-known-issues310.md:74]: We should change all occurrences of "Web UI" to "web interface".
Does this refer to the traditional View type only?
```suggestion
| **Date Added:** 2022-09-29 <br> **Component:** Web UI <br> **Deployment Mode:** All <br> **Description:** In the web interface for Views of the `arangosearch` type, it's not possible to create a link to a collection, if the collection has only one letter in its name. <br> **Affected Versions:** 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [BTS-1008](https://arangodb.atlassian.net/browse/BTS-1008){:target="_blank"} (internal) |
```

**Review by Simran-B — CHANGES_REQUESTED** [review]

File: 3.10/release-notes-known-issues310.md [hunk 3]
```diff
@@ -67,4 +70,6 @@ Other
...
+| **Date Added:** 2022-09-29 <br> **Component:** ArangoDB Starter <br> **Deployment Mode:** All <br> **Description:** ArangoDB Starter may fail to pick a docker container name from cgroups. <br> **Affected Versions:** 3.8.x, 3.9.x, 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [GT-207](https://arangodb.atlassian.net/browse/GT-207){:target="_blank"} (internal) |
```
Comment (Simran-B) [inline 3.10/release-notes-known-issues310.md:75]:
```suggestion
| **Date Added:** 2022-09-29 <br> **Component:** ArangoDB Starter <br> **Deployment Mode:** All <br> **Description:** The ArangoDB Starter may fail to pick a Docker container name from cgroups. <br> **Affected Versions:** 3.8.x, 3.9.x, 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [GT-207](https://arangodb.atlassian.net/browse/GT-207){:target="_blank"} (internal) |
```

File: 3.10/release-notes-known-issues310.md [hunk 4]
```diff
@@ -67,4 +70,6 @@ Other
...
+| **Date Added:** 2022-09-29 <br> **Component:** Web Interface <br> **Deployment Mode:** All <br> **Description:** In the web interface for ArangoSearch Views, it's not possible to create a link to a collection, if the collection has only one letter in its name. <br> **Affected Versions:** 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [BTS-1008](https://arangodb.atlassian.net/browse/BTS-1008){:target="_blank"} (internal) |
+| **Date Added:** 2022-09-29 <br> **Component:** ArangoDB Starter <br> **Deployment Mode:** All <br> **Description:** The ArangoDB Starter may fail to pick a Docker container name from cgroups. <br> **Affected Versions:** 3.8.x, 3.9.x, 3.10.x <br> **Fixed in Versions:** - <br> **Reference:** [GT-207](https://arangodb.atlassian.net/browse/GT-207){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.10/release-notes-known-issues310.md:75]: I suppose GT-207 should be mentioned in 3.8, 3.9 and 3.11 also.

**Review by KVS85 — APPROVED** [review]

**Pull request merged**